server/routes/contacts/manage/list/listContactsController.test.ts

```diff
@@ -56,15 +58,87 @@ afterEach(() => {
 describe('GET /contacts/manage/list', () => {
   it('should render the contacts list for a prisoner', async () => {
     auditService.logPageView.mockResolvedValue(null)
+    const contactsList: PrisonerContactSummary = [
+      {
+        prisonerContactId: 100,
+        contactId: 200,
+        prisonerNumber: 'G9381UV',
+        surname: 'Adams',
+        forename: 'Claire',
+        middleName: '',
+        dateOfBirth: new Date('1973-01-10'),
+        relationshipCode: 'code here',
+        relationshipDescription: 'Friend',
+        flat: 'FLAT 1',
+        property: 'Proeperty',
+        street: '123 High Street',
+        area: 'Mayfair',
+        cityCode: 'London',
+        countyCode: 'county code here',
+        postCode: 'W1 1AA',
+        countryCode: 'England',
+        approvedVisitor: true,
+        nextOfKin: true,
+        emergencyContact: true,
+        awareOfCharges: true,
+        comments: 'comments here',
+      },
+    ]
 
-    // TODO: Tim
-    // prisonerSearchService mock result
-    // contactsService mock result
+    prisonerSearchService.getByPrisonerNumber.mockResolvedValue(TestData.prisoner())
+    contactsService.getPrisonerContacts.mockReturnValue(contactsList)
 
     const response = await request(app).get(`/contacts/manage/list/${journeyId}`)
 
+    // Prisoner
     expect(response.status).toEqual(200)
+    expect(response.text).toContain('John')
+    expect(response.text).toContain('Smith')
+    expect(response.text).toContain('2 April 1975')
```

Comment:
We could be a bit more specific about which fields/columns contain which values?

Reply:
I just added `cheerio` which makes this a bit easier as you can use jquery type selectors.

```
    const $ = cheerio.load(response.text)
    expect($('#firstName').val()).toStrictEqual('first')
    expect($('#middleName').val()).toStrictEqual('middle')
    expect($('#lastName').val()).toStrictEqual('last')
    expect($('#title').val()).toStrictEqual('MR')
```

Reply:
changes added using cheerio